Getting-Started.md

```diff
@@ -166,61 +149,62 @@ upstream	https://github.com/chaoss/grimoirelab-perceval (push)
 
 In order to add a remote to a Git repository, you can use the following command:
 ```
-~/sources/perceval$ git remote add upstream https://github.com/chaoss/grimoirelab-perceval
+$ git remote add upstream https://github.com/chaoss/grimoirelab-perceval
 ```
 
-Then, you can download the [PyCharm community edition](https://www.jetbrains.com/pycharm/download/#section=linux), using any of the following alternatives:
-
-* **Alternative 1: Install PyCharm via Snap Package Management**
-    
-    Install Snap Package Management using the below command.
-
-     `sudo apt install snapd snapd-xdg-open`
-    
-    Now that `Snap` is installed, run the command below to install the community version of PyCharm.
-    
-     `sudo snap install pycharm-community --classic`
-
-* **Alternative 2: Install PyCharm via Ubuntu Software Center**
-
-    Open Ubuntu Software Center and search for PyCharm. Then, select and install the community version of PyCharm.
-
-* **Alternative 3: Download PyCharm tar file**
+#### ProTip [&uarr;](#cloning-the-repositories-)
 
-    Go to the official website of [JetBrains]((https://www.jetbrains.com/pycharm/download/#section=linux)) and download the community version of PyCharm for your system.
-    Then, extract the downloaded `tarfile` file using the below command.
-    
-    `tar -xzf tarfile`
+You can use this use this [script]() to automate this whole process.
```

Comment:
I have left this link, for now. 

I have made a script for it. I am waiting for it to be reviewed. It would be really great if you can help me with testing it. You can find the info here, https://github.com/vchrombie/grimoirelab-scripts/issues/1.

Thanks in advance.

Reply:
Thanks to @abhiandthetruth @animeshk08 and @valeriocos for testing and reviewing the script. I have added the link now. :tada: